src/Symfony/Bundle/SecurityBundle/DependencyInjection/MainConfiguration.php

```diff
@@ -386,9 +386,9 @@ private function addEncodersSection(ArrayNodeDefinition $rootNode)
             ->children()
                 ->arrayNode('encoders')
                     ->example(array(
-                        'Acme\DemoBundle\Entity\User1' => 'sha512',
+                        'Acme\DemoBundle\Entity\User1' => 'bcrypt',
                         'Acme\DemoBundle\Entity\User2' => array(
-                            'algorithm' => 'sha512',
+                            'algorithm' => 'bcrypt',
                             'encode_as_base64' => 'true',
```

Comment:
We need to remove `encode_as_base64` and `iterations` ... but we could add `cost`